### PR TITLE
Form feature branch

### PR DESCRIPTION
**Pull request by joeycsea**
Hi Jay!
Please review the code and let me know if you have any questions!
Thanks 😊

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+import Header from './Header';
+import Form from './Form.js';
+import DisplayWeather from './DisplayWeather';
+import Footer from './Footer.js';
 
 function App() {
   return (
     <div className="App">
-
+      <Header />
```
Comment (jhshim85) [inline src/App.js:10]: Great setup with components!

File: src/DisplayWeather.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// DisplayWeather component
+
+const DisplayWeather = () => {
+   return (
+      <section>
```
Comment (jhshim85) [inline src/DisplayWeather.js:5]: instead of user chooses what's to display, are we displaying all types of weather?

File: src/Footer.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Footer component
+
+const Footer = () => {
+   return (
+      <footer>
+         <p>Made by <a href='https://www.joeysea.com/' target='_blank' rel='noreferrer'>Joey Sea</a> and <a href='https://www.jaehyunshim.com/' target='_blank' rel='noreferrer'>Jaehyun(Jay) Shim</a></p>
```
Comment (jhshim85) [inline src/Footer.js:6]: well done!

File: src/Form.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+// Form component
+import { useState } from "react";
+
+
+const Form = () => {
+
+   const [ cityChoice, setCityChoice ] = useState("");
```
Comment (jhshim85) [inline src/Form.js:7]: great use of state hook!

File: src/Form.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            <input 
+               placeholder="Toronto"
+               required
+               type="text" 
+               id="city" 
+               name="city" 
+               onChange = {handleCityChange} 
+               value = {cityChoice} 
+            />
+         <button type="submit">Search</button>
```
Comment (jhshim85) [inline src/Form.js:31]: I thought we were giving the users a choice of which type of weather they want it to be displayed. (current, hourly, 10 days)
we can either do select or radio.
let me know what you think ^^
I don't mind just displaying all three~

**Review by jhshim85 — COMMENTED** [review]
great setup for our project!
please give a second thought on how we want to display the weather. (all types or just choose one)
or we can display all and use then routing to display only the one that users choose/select after
let me know ^^

File: src/DisplayWeather.js (hunk 2 above)
Comment (joeycsea) [reply src/DisplayWeather.js:5]: We will display Current Weather at all times. I will include 2 buttons that will route to Hourly Forecast and 10 Day Forecast.

**Pull request closed without merging**